appmap/test/normalize.py

```diff
@@ -46,6 +46,13 @@ def normalize_metadata(metadata):
                 v = f.pop('version')
                 assert v == pytest.__version__
 
+def normalize_headers(dct):
+    """Remove some headers which are variable between implementations.
+    This allows sharing tests between web frameworks, for example.
+    """
+    for hdr in ['User-Agent', 'Content-Length', 'ETag', 'Cookie', 'Host']:
+        dct.pop(hdr, None)
```

Comment:
You were going to add a check for the type of the value here, weren't you?

Reply:
Right, it's in the other PR: 54f64eebe64854a25d623774509306d0f68bd560; I cherry-picked an older version here unintentionally. While merging the newer version will prevail :)